### ISSUE TITLE
[1.12.2]  Weather effects not working when Harvest Festival Legacy is installed

### ISSUE THREAD

**Issue by ghost** [OreCruncher/DynamicSurroundings#782]
**Mod Version:**
DynamicSurroundings-1.12.2-3.6.3
Harvest-Festival-Legacy-1.12.2-0.6.23
(though, tested on multiple versions for both)

**Forge Version:**
Forge - 14.23.5.2854

This issue has persisted on basically every mod version of both mods.  When Dynamic Surroundings and Harvest Festival Legacy are both installed, Dynamic Surroundings' weather effects (Nether effects, dust storms and water ripple) don't work at all.  No crashes or anything else unusual in the log, apart from a small message from Harvest Festival (Not hooking weather renderer for dimension) after I created, joined the world and immediately set the weather to rain.  This message did not appear when I launched without Dynamic Surroundings.

[09:07:46] [Server thread/INFO]: elimang joined the game
[09:07:46] [Client thread/INFO]: Attaching capabilities to world [overworld] (CLIENT)
[09:07:46] [Client thread/INFO]: DimensionInfo{id=0, name=overworld, seaLevel=63, cloudHeight=128, skyHeight=256, haze=true, aurora=true, weather=true, fog=true}
[09:07:46] [Client thread/INFO]: Creating default SeasonInfo for dimension overworld
[09:07:46] [Client thread/INFO]: Not hooking weather renderer for dimension [overworld] (class joshie.harvest.calendar.render.WeatherRenderer)

**Comment by OreCruncher** [comment]
Correct.  There is another mod that hooks the weather renderer so DS will defer to that one.  When that happens you will not get the DS weather effects because it is expected that another mod will be providing that.  From the log it looks like its some sort of harvest mod that has a calendar (joshie.harvest.calendar.render.WeatherRenderer).  FWIW there can only be a single weather renderer active.
